config/serverless.yml

```diff
@@ -152,3 +152,4 @@ xpack.reporting.statefulSettings.enabled: false
 
 # Disabled Observability plugins
 xpack.ux.enabled: false
+xpack.monitoring.enabled: false
```

Comment:
I checked using `oblt` and it's still visible in the search  🤔 We can put `monitoring.ui.enabled: false` here or inside `serverless.oblt.yml`. Maybe having it here will make more sense instead of having it in both `es` and `oblt`

Reply:
Thanks :)